src/themes/components/input.ts

```diff
@@ -16,23 +13,19 @@ const baseStyle: PartsStyleObject<typeof parts> = {
     appearance: 'none',
     transitionProperty: 'common',
     transitionDuration: 'normal',
+    px: 'md',
+    py: 'xs',
   },
 };
 
 const size: Record<string, SystemStyleObject> = {
   md: {
-    fontSize: 'lg',
```

Comment:
I was checking the input component in figma -> https://www.figma.com/file/CHU4fpMfXpiAW4qNFzi9y1/AS24-WebComponents?node-id=55%3A3221

And it seems that we did wrong here with the font sizes. 
Md: fontSite = 14px
Md: fontSite = 16px

We were having `lg` for both variation but the `...typography.body,` was rewriting those values.

Please doble check the input and let me know if these changes make sense. Thank you!
We are cleaning, cleaning and adjusting wuiii 🧼🧼🧼🧼

Reply:
Could you double-check this with a designer? The example with just a placeholder has `14px` the example with a value uses `16px` so I'm not sure anymore which one is correct.

Reply:
@Averethel I double checked with Enrique and the placeholder has 16px and with value 16px as well when the height input is 48px. If the height input is 36px the placeholder and the value texts have 14px. It was a mistake in the figma but Enrique already fix it.